package.json

```diff
@@ -14,9 +14,11 @@
     "@fontsource/roboto": "^4.5.7",
     "@mui/icons-material": "^5.8.4",
     "@mui/material": "^5.9.2",
+    "@react-oauth/google": "^0.2.8",
     "axios": "^0.27.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-google-login": "^5.2.2",
```

Comment:
Remove this dependency: `react-google-login`. (just remove the line)

Reply:
fixed with new commit